api/v1alpha1/integrationtestscenario_conversion.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2023.
```

Comment:
Why this was changed to 2022? It was created in 2023

Reply:
git log --reverse shows intial commit and initial code as May 10th 2022

Reply:
I remember doing reviews for it this year
```
git log api/v1alpha1/integrationtestscenario_conversion.go

commit 471556e4bf32d3cfcadf2151da19c4277dfbc904
Author: Jiri Sztuka <jsztuka@redhat.com>
Date:   Thu Apr 13 21:00:13 2023 +0200

    feat(STONEINTG-386): mutating webhook
    
    Signed-off-by: Jiri Sztuka <jsztuka@redhat.com>

```

It has only one commit from April this year.

This file was created in 2023, why it should have 2022?